package.json

```diff
@@ -6,7 +6,9 @@
   "main": "build/nv.d3.js",
   "scripts": {
     "test": "grunt",
-    "build": "grunt"
+    "test-unit": "mocha test/mocha/*.js",
+    "build": "grunt",
+    "stryker": "stryker run stryker.conf.js"
```

Comment:
Please remove build related things from this PR and keep it just the new model and examples.